jsonschema-core.xml

```diff
@@ -2404,7 +2404,7 @@
                         </t>
                         <t>
                             The annotation result of this keyword is the set of instance
-                            property names matched by this keyword.
+                            property names which are also present in this keyword.
```

Comment:
I think "under" more fits language we use elsewhere, rather than "in"?

```suggestion
                            property names which are also present under this keyword.
```